db/db.py

```diff
@@ -0,0 +1,28 @@
+import pymongo
+
+class db:
```

Comment:
Class names should have an upper case first letter.
Also it would be great to write the name out ("Database")